content/docs/user-guide/assets/asset-processor/asset-cache-server.md

```diff
@@ -22,77 +22,151 @@ The next sections assume the root folder to be `T:/o3de` but it can be in any fo
+
+![Create a log tab in Asset Processor](/images/user-guide/assets/asset-processor/acs_snapshot.png)
+
+### Choose a Mode
+
+The Shared Cache mode can be set to:
+ 1. Inactive - The Shared Cache is not active. All the assets will be processed locally.
+ 2. Server - The Asset Processor will archive product assets on a remote folder.
+ 3. Client - The Asset Processor will attempt to retrieve archive product assets from a remote folder.
+ 
+The system defaults to Inactive.
+
+### Select a Transfer Directory
+
+This field should have the full path to the remote folder that was set up in the "Create the transfer directory" step. When the "Save Changes" button is clicked, the system will detect the validity of the transfer directory.
+
+### Manage Shared Cache Patterns
+
+The table shows the active asset patterns that will be cached when the asset is processed on the server. The asset pattern row contains:
+ 1. a check box to enable or disable the pattern
+ 2. a text box to store the pattern's name
+ 3. a combo box how to use the pattern to match source assets (Wildcard or RegEx)
+ 4. a text box to enter the pattern to match source assets
+ 5. a button to remove the pattern
+ 
+The enable flag allows a user to toggle the row to disabled to force the local client Asset Processor to process the matching assets locally. Most of the time the client users will have all patterns enabled.
+
+The `Name` is a label to explain what source assets are being matched. If the name matches the name of an asset builder, the source assets that use the asset builder will cache the product archives.
+
+The `Type` is either `Wildcard` or `RegEx` and tells the system how to use the `Pattern` text to match source asset files. `Wildcard` is normally starts with an asterisk (\*) and is followed by a dot (.) and an extension to match all source assets with a certain extension such as `*.png` and `*.wav` to match PNG and WAV files, respectively. The `RegEx` type interprets the `Pattern` as a regular expression so that projects can specify more advanced matching such as sub-folders or multiple asset types. 
+
+The trash can button can be used to remove an existing asset pattern row. The "+ Add Pattern" can be used to add a new asset pattern row.
+
+### Save or Discard Buttons
+
+The `Save Changes` and `Discard` buttons become enabled when a change to the settings has been detected. The `Save Changes` commits the changes to a settings file. The `Discard` button reverts the panel's values.
+
+The settings file is written to the `{project_folder}/Registry/asset_cache_server_settings.setreg` file. 
+
+If the Remote Folder is set to an invalid folder location, then an error dialog will show up and no settings will be written.
+
+# Asset Cache Server: Settings Details
+
+The settings for the Asset Cache Server are held in `.setreg` files inside the project. For example the `o3de/AutomatedTesting/Registry/editorpreferences.setreg` stores the settings for the Editor.
```

Comment:
You might want to clarify that those are just the settings for the Editor for the Automated Testing project, and not other projects.